src/NorthwindWebsite/API/Controllers/CategoriesController.cs

```diff
@@ -18,6 +18,10 @@ public CategoriesController(ICategoryService categoryService)
         _categoryService = categoryService;
     }
 
+    /// <summary>
+    /// Returns all categories from database.
+    /// </summary>
+    /// <returns><see cref="Task">Representing an asynchronous operarion.</returns>
```

Comment:
What about `response` attribute?
This comment is relevant for the other controller actions as well